bijou.js

```diff
@@ -55,6 +55,8 @@ Contributors to Bijou.js:
 ╠═══════════════════╬════════════════════════════╣
 ║ YOYITsM3M8        ║ Suggested a lot of         ║
 ║ (AKA retronbv)    ║ features and bug fixes.    ║
+║═══════════════════║════════════════════════════║
+║ thecoder876       ║ Made some improvements.     ║
```

Comment:
Hmmmmmm; This doesn't explain much, could you choose something like "web design: